arbitrum-docs/launch-orbit-chain/concepts/custom-gas-token-sdk.md

```diff
@@ -18,11 +18,13 @@ import PublicPreviewBannerPartial from '../partials/_orbit-public-preview-banner
 
 ### Custom gas token SDK
 
-The following custom gas token APIs are available under the `orbit-custom-fee-token` SDK tag:
+The `orbit-custom-fee-token` SDK tag offers a range of custom gas token APIs. These are primarily used for **bridging** between the parent chain and a custom gas token orbit chain. If you're looking to bridge a native token or an ERC20 token from the parent chain to an orbit chain that uses a `custom gas token`, you'll need to first authorize the native token on the parent chain. Our APIs make this process easier by enabling token approval and providing a way to check the current approval status.
 
 - `getApproveFeeTokenRequest` and `approveFeeToken` on [EthBridger](https://github.com/OffchainLabs/arbitrum-sdk/pull/310/files#diff-a977cd005aca51be6f05bc7e1c7c1bf6d734b62b2c45c84b05e2eb0c3c3c6fff)
 - `getApproveFeeTokenRequest` and `approveFeeToken` on [Erc20Bridger](https://github.com/OffchainLabs/arbitrum-sdk/pull/310/files#diff-b1894b842df6f4794b6623dc57e9e14c2519fbe5fa5c5dd63403f1185f305cbb)
 
 If you're using a custom gas token, you'll need to use each of these; on custom gas token chains, token approval is required when transferring ERC20 tokens (while Orbit chains that use ETH do not have this requirement).
```

Comment:
```suggestion
If you're depositing into a chain with a custom gas token, you'll need to use the relevant helper method; on custom gas token chains, the token must be approved on the parent chain in order to pay for the cross chain transaction carrying the deposit while Orbit chains that use ETH do not have this requirement.
```

I think it'd be good to clean up this sentence a bit too. Reads to me as though it's saying that approvals are needed for transferring on the chain rather than when depositing.